projects/biogears/libBiogears/src/engine/Systems/Environment.cpp

```diff
@@ -745,7 +752,7 @@ void Environment::CalculateEvaporation()
 
     for (SEThermalCircuitPath* envSkinToGround : m_EnvironmentSkinToGroundPaths) {
       double dClothingResistance_clo = GetConditions().GetClothingResistance(HeatResistanceAreaUnit::clo) * m_cloSegmentation[index];
-      double dClothingResistance_m2_kPa_Per_W = clo_To_m2_K_Per_W * dClothingResistance_clo / (iCl * dLewisRelation_K_Per_kPa);
+      double dClothingResistance_m2_kPa_Per_W = 100000*clo_To_m2_K_Per_W * dClothingResistance_clo / (iCl * dLewisRelation_K_Per_kPa);
```

Comment:
Major multiplier batman. I have this very basic request that we start using digit separators.  `100_000`

Reply:
yeah this is sort of a shallow patch for now to have something happen due to clothing change. biogears currently doesn't do pretty much anything when you take all your clothes off

Reply:
i'll be working more on hypothermia over the coming 6 months so I'm hoping to update the models to be a bit more robust lol